x/leverage/spec/README.md

```diff
@@ -0,0 +1,41 @@
+# Leverage Module
+
+## Abstract
+
+This document specifies the `x/leverage` module of the Umee chain.
+
+The leverage module allows users to lend and borrow assets, and implements various features to support this, such as a token accept-list, a dynamic interest rate module, incentivized liquidation of undercollateralized debt, and automatic reserve-based repayment of bad debt.
+
+The leverage module depends directly on `x/oracle` for asset prices, and interacts indirectly with `x/ibctransfer`, `x/peggy`, and the cosmos `x/bank` module as these all affect account balances.
+
+## Contents
+
+1. **[Concepts](01_concepts.md)**
+    - [Accepted Assets](01_concepts.md#Accepted-Assets)
+        - [uTokens](01_concepts.md#uTokens)
+    - [Lending and Borrowing](01_concepts.md#Lending-and-Borrowing)
+    - [Reserves](01_concepts.md#Reserves)
+    - [Liquidation](01_concepts.md#Liquidation)
+    - Important Derived Values:
+        - [uToken Exchange Rate](01_concepts.md#uToken-Exchange-Rate)
+        - [Borrow Utilization](01_concepts.md#Borrow-Utilization)
+        - [Borrow Limit](01_concepts.md#Borrow-Limit)
+        - [Borrow APY](01_concepts.md#Borrow-APY)
+        - [Lending APY](01_concepts.md#Lending-APY)
+        - [Close Factor](01_concepts.md#Close-Factor)
+        - [Market Size](01_concepts.md#Market-Size)
```

Comment:
I would like to ask in advance whether these sections belong in the spec or are safe being only in the ADRs.

They're all written but can be trimmed down or removed if not necessary.

Reply:
@toteki not totally sure I follow the question?

Reply:
Was mainly wondering if I went into more detail than necessary at the end of `01_concepts.md`. If it doesn't look like a problem then nothing needs to be done there.